packages/ui/src/components/InputField/index.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames';
+import type { ForwardedRef, HTMLProps, ReactElement } from 'react';
+import { forwardRef, cloneElement } from 'react';
+
+import type { ErrorType } from '@/components/ErrorMessage';
+import ErrorMessage from '@/components/ErrorMessage';
+
+import * as styles from './index.module.scss';
+
+type Props = HTMLProps<HTMLInputElement> & {
+  className?: string;
+  error?: ErrorType;
+  isDanger?: boolean;
```

Comment:
Is this equivalent to `hasError`?

Reply:
yes, aligned with design styling,  'danger'

Reply:
Also, in some cases, the error is not empty but should not mark the input field  `hasError`